cscs-checks/system/slurm/slurm.py

```diff
@@ -227,9 +223,18 @@ def __init__(self):
                 ),
             }
         }
-        # }}}
 
-    # {{{ hooks
+    @performance_function('GB')
+    def max_cn_memory(self):
+        return sn.getattr(self, 'reference_meminfo')
```

Comment:
I don't know why we need this? Is it just for recording the reference meminfo?

Reply:
I think so, but I'm not sure. Maybe it comes from a time where it wasn't possible to put the reference on the perflog? I think we should remove it. What do you think @victorusu?